backends-common/rabbitmq/src/main/java/org/apache/james/backend/rabbitmq/CassandraMailQueueViewConfiguration.java

```diff
@@ -17,11 +17,13 @@
  * under the License.                                           *
  ****************************************************************/
 
-package org.apache.james.queue.rabbitmq.view.cassandra.configuration;
+package org.apache.james.backend.rabbitmq;
```

Comment:
I DISAGREE with this move.

That have nothing to do with `backend` and this code is not generic but specific to the mail queue.